.github/workflows/feature-branch.yml

```diff
@@ -29,6 +29,17 @@ jobs:
     secrets:
       ECR_ROLE_TO_ASSUME: ${{ secrets.ECR_ROLE_TO_ASSUME }}
 
+  scan-image:
```

Comment:
Is it worth adding Trivy somewhere in the name?

Reply:
Done, renamed to `trivy-image-scan`